lib/merge-yaml

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3
+
+import json
+import hiyapyco
+import click
```

Comment:
This is also not installed. And since the options are not very complex here, IMO it isn't worth installing, but simply use argparse from standard library.